src/Services/QueuedJobService.php

```diff
@@ -519,6 +519,26 @@ public function checkJobHealth($queue = null)
             ]
         );
 
+        // Send broken job email notification
+        $subject = _t(__CLASS__ . '.BROKEN_JOBS', 'Broken job(s)');
+        $message = htmlentities(sprintf(_t(
+            __CLASS__ . '.BROKEN_JOBS_MSG',
+            implode(' ', [
+                'The following job(s) appear to be broken.',
+                'Log in to %s to see further details and take any necessary actions.',
+            ])
+        ), Director::absoluteBaseURL()));
+        foreach ($brokenJobs as $brokenJob) {
+            // Using htmlentities() because of the use of $Message.RAW in QueuedJobsBrokenJobs.ss
+            // in order for the <br> tags to render as HTML
+            $message .= sprintf("<br>- %s (ID: %s)", htmlentities($brokenJob->JobTitle), $brokenJob->ID);
+        }
```

Comment:
```suggestion
        $message = 'The following job(s) appear to be broken.';
```
You can use `_t` if you want - the point is it should just be the message part, not the full template.

Reply:
Updated - though I've retained most of the message here so that's all translatable